Makefile

```diff
@@ -112,14 +68,19 @@ checklicense:
 	addlicense -check -f ./hack/license.txt $(shell find . -path ./hack/tools/vendor -prune -false -o -name *.sh)
 	addlicense -check -f ./hack/license.txt $(shell find . -path ./hack/tools/vendor -prune -false -o -name Dockerfile)
 
-.PHONY: example-deploy-nginx
-example-deploy-nginx:
-	kubectl --kubeconfig ./shared-services.kubeconfig apply -f ./manifests/example/nginx/certs.yaml
-	kubectl --kubeconfig ./shared-services.kubeconfig apply -f ./manifests/example/nginx/nginx.yaml
-	kubectl --kubeconfig ./shared-services.kubeconfig apply -f ./manifests/example/nginx/exported_http_proxy.yaml
-
-.PHONY: example-curl-nginx
-example-curl-nginx:
-	kubectl --kubeconfig ./workloads.kubeconfig run -it --rm --restart=Never \
-		--image=curlimages/curl curl -- \
-		curl -v -k "https://nginx.xcc.test"
+# find or download controller-gen
+# download controller-gen if necessary
+controller-gen:
+ifeq (, $(shell which controller-gen))
```

Comment:
This was a straight copy-paste from the kubebuilder Makefile. I'm partial to leaving it because it seems like it is useful even though it isn't consistently applied to all our binaries in the Makefile, but don't feel too strongly. If you feel like the inconsistency is an issue we can remove it.